cocodevicesdk/examples/c/brightness-device-app/CMakeLists.txt

```diff
@@ -47,13 +55,12 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pedantic-errors -Wall -std=c11 -D_GNU_SOURC
 file(GLOB SRCS *.c *.h)
 
 # Add final executable for this folder along with library dependencies
-add_executable(deviceapp ${SRCS})
+add_executable(deviceapp ${SRCS} ${OBJS})
 
 # Specify the libraries to be linked dynamically
 if(${APPLE})
-  target_link_libraries(deviceapp libcocodevicesdk_static.a)
-else()
-  target_link_libraries(deviceapp cocodevicesdk)
+  target_link_libraries(cococlientsdk "-framework CoreFoundation")
```

Comment:
target cococlientsdk? copy paste error?

also I wonder if this compiles on macos or not. Will try it tomorrow.

Reply:
yes, apparently missed it, copy-paste 😅. Actually we should be using ${PROJECT_NAME} instead of repeating the name over and over.